Gruntfile.js

```diff
@@ -31,13 +28,16 @@ module.exports = function ( grunt ) {
 	grunt.loadTasks( "tasks" );
 
 	// These plugins provide necessary tasks
-	grunt.loadNpmTasks( "grunt-contrib-jshint" );
+	grunt.loadNpmTasks( "grunt-eslint" );
 	grunt.loadNpmTasks( "grunt-contrib-clean" );
 	grunt.loadNpmTasks( "grunt-contrib-nodeunit" );
 
-	// Whenever the "test" task is run, first clean the "tmp" dir, then run the test.
+	// Whenever the "test" task is run,
```

Comment:
Multi-line comments should use the proper comment notation (i.e. use block comments).